src/lib/modules/contracts_manager/index.js

```diff
@@ -118,11 +118,46 @@ class ContractsManager {
             try {
               const gas = await contractObj.methods[req.body.method].apply(this, req.body.inputs).estimateGas();
               contractObj.methods[req.body.method].apply(this, req.body.inputs)[funcCall]({from: account, gasPrice: req.body.gasPrice, gas: Math.floor(gas)}, (error, result) => {
+                const paramString = abi.inputs.map((input, idx) => {
+                  const quote = input.type.indexOf("int") === -1 ? '"' : '';
+                  return quote + req.body.inputs[idx] + quote;
+                }).join(', ');
+
+                let contractLog = {
+                  name: req.body.contractName,
+                  functionName: req.body.method,
+                  paramString: paramString,
+                  address: contract.deployedAddress,
+                  status: '0x0'
+                };
+
                 if (error) {
+                  self.events.emit('contracts:log', contractLog);
                   return res.send({result: error.message});
                 }
 
-                res.send({result});
+                if(funcCall === 'call') {
```

Comment:
How do you feel about putting `call` in a more descriptive constant?

```
if (funcCall === WEB3_CONTRACT_METHOD_CALL) {
...
}
```

Or something along those lines..